app/2.0/docs/devguide/hybrid-elements.md

```diff
@@ -251,13 +250,17 @@ Other sets are listed in a special `variants` section. For example:
   },
 ```
 
-In the example above, the default dependencies match either 1.x or 2.x, while the 1.x variant
+In the example above, the default dependencies match 2.x, while the 1.x variant
```

Comment:
I think this should be "default dependencies work with either Polymer 1.x or 2.x"